src/buy/components/BuyMessage.tsx

```diff
@@ -3,16 +3,25 @@ import { useBuyContext } from './BuyProvider';
 
 export function BuyMessage() {
   const {
-    lifecycleStatus: { statusName },
+    lifecycleStatus: { statusName, statusData },
   } = useBuyContext();
 
-  if (statusName !== 'error') {
-    return null;
+  // Missing required fields
+  if (statusName === 'error' && statusData.code === 'TmBPc05') {
```

Comment:
any reason to not just always show the error.message on errors?

Reply:
in other components we only show `Something went wrong...` so i'm keeping consistent for now and will investigate better errors in follow up but it might be something we want to change for all our components.